tools/sense_studio/sense_studio.py

```diff
@@ -255,6 +272,38 @@ def project_details(path):
     return render_template('project_details.html', config=config, path=path, stats=stats)
 
 
+@app.route('/add-class/<string:project>', methods=['POST'])
+def add_class(project):
+    """
+    Add a new class to the given project.
+    """
+    # Lookup project path
+    projects = _load_project_overview_config()
+    path = projects[project]['path']
+
+    # Get class name and tags
+    data = request.form
+    class_name = data['class']
+    tag1 = data['tag1'] or f'{class_name}_tag1'
+    tag2 = data['tag2'] or f'{class_name}_tag2'
+
+    if tag2 == tag1:
+        tag2 = f'{tag2}_2'
```

Comment:
If I input `test5_1` as the tag name for both tags, then the second tag would be renamed to `test5_1_2`. But wouldn't it then make sense to rename the first one as `test5_1_1` as well? :thinking:

Reply:
We could do that :+1: